src/NServiceBus.AcceptanceTesting/ScenarioWithContext.cs

```diff
@@ -80,45 +59,72 @@ public Task<IEnumerable<TContext>> Run(TimeSpan? testExecutionTimeout = null)
 
             Console.WriteLine("Total time for testrun: {0}", sw.Elapsed);
 
-            return runDescriptors.Select(r => (TContext)r.ScenarioContext);
+            return runDescriptors.Select(r => (TContext) r.ScenarioContext);
         }
 
-        public IAdvancedScenarioWithEndpointBehavior<TContext> Repeat(Action<RunDescriptorsBuilder> action)
+        public IAdvancedScenarioWithEndpointBehavior<TContext> MaxTestParallelism(int maxParallelism)
         {
-            runDescriptorsBuilderAction = action;
+            limitTestParallelismTo = maxParallelism;
 
             return this;
         }
 
-        public IScenarioWithEndpointBehavior<TContext> AllowExceptions(Func<Exception, bool> filter = null)
+        public IAdvancedScenarioWithEndpointBehavior<TContext> Should(Action<TContext> should)
         {
-            if (filter == null)
+            shoulds.Add(new ScenarioVerification<TContext>
             {
-                filter = exception => true;
-            }
+                ContextType = typeof(TContext),
+                Should = should
+            });
 
-            allowedExceptions = filter;
             return this;
         }
 
-        public IScenarioWithEndpointBehavior<TContext> AllowSimulatedExceptions()
+        public IAdvancedScenarioWithEndpointBehavior<TContext> Report(Action<RunSummary> reportActions)
         {
-            return AllowExceptions(e => e is SimulatedException);
+            reports = reportActions;
+            return this;
         }
 
-        public IAdvancedScenarioWithEndpointBehavior<TContext> MaxTestParallelism(int maxParallelism)
+        public IScenarioWithEndpointBehavior<TContext> WithEndpoint<T>() where T : EndpointConfigurationBuilder
         {
-            limitTestParallelismTo = maxParallelism;
+            return WithEndpoint<T>(b => { });
+        }
+
+        public IScenarioWithEndpointBehavior<TContext> WithEndpoint<T>(Action<EndpointBehaviorBuilder<TContext>> defineBehavior) where T : EndpointConfigurationBuilder
+        {
+            var builder = new EndpointBehaviorBuilder<TContext>(typeof(T));
+
+            defineBehavior(builder);
+
+            behaviors.Add(builder.Build());
+
+            return this;
+        }
+
+        public IScenarioWithEndpointBehavior<TContext> Done(Func<TContext, bool> func)
+        {
+            done = c => func((TContext) c);
```

Comment:
Same formatting issue, or is this something that's intentionally different from default VS settings?